src/config_server/serverConfig.ts

```diff
@@ -0,0 +1,38 @@
+import tedious, { ConnectionConfiguration, Connection } from 'tedious';
+import { PORT, server } from '../app';
+
+export let connection: Connection;
+
+export const runServerConfiguration = () => {
```

Comment:
Let's make this name say something about the DB so we know what server we're talking about.